src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -81,8 +84,9 @@ public boolean is(CloudPlatform cloudPlatform) {
 
   public abstract CloudPlatform getCloudPlatform();
 
-  public abstract List<? extends StorageResource<?, ?>> createStorageResourceValues(
-      DatasetRequestModel datasetRequest);
+  public abstract <T extends CloudResource, U extends CloudRegion>
```

Comment:
This and other changes below were added to fix sonar warnings. In this case sonar was warning that wildcard types were being used, and more specific types should be used instead.